test/include/helpers/sceneHelpers.hpp

```diff
@@ -52,3 +53,27 @@ static inline void setupBoxesAlongAxes()
 		spawnCubeOnScene(Mat3x4f::TRS({0, 0, (2 * scaleZ + 2) * i}, {0, 0, 45}, {1, 1, scaleZ}));
 	}
 }
+
+static inline rgl_mesh_t loadFromSTL(const char* path)
+{
+	std::vector<float> vertices, _normals;
+	std::vector<int32_t> indices, _solids;
```

Comment:
Do names beginning with underscore fall under our naming convention?

Reply:
To my knowledge, this is widely used idiom to mark that the variable is intentionally not used.